### PR TITLE
test(atomic): add cypress test for breadbox

### PR DESCRIPTION
**Pull request by lvu285**
KIT-930
- Add breadcrumb for each facet _ but not rating & rating range
- Add general breadcrumb tests with ShowMore, ShowLess, ClearAll

### CODE REVIEW

**Review by btaillon — APPROVED** [review]
Looks good!

I think you should fix the [linter errors in your changed files](https://github.com/coveo/ui-kit/pull/1631/files) though. Don't bother with linter errors that aren't from files you haven't changed, I fixed them [here](https://github.com/coveo/ui-kit/pull/1627).

File: packages/atomic/cypress/integration/breadbox/breadbox-actions.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {addTag, TagProps, TestFixture} from '../../fixtures/test-fixture';
+import {BreadboxSelectors} from './breadbox-selectors';
+
+export const breadboxLabel = 'Filters';
+export const addBreadbox =
+  (props: TagProps = {}) =>
+  (env: TestFixture) =>
+    addTag(env, 'atomic-breadbox', props);
+
+export function unselectBreadcrumbAtIndex(index: number) {
```
Comment (ThibodeauJF) [inline packages/atomic/cypress/integration/breadbox/breadbox-actions.ts:10]: nitpick: replace unselect with deselect (in other places too)

File: packages/atomic/cypress/integration/facets/facet/facet.cypress.ts [hunk 2]
```diff
@@ -912,4 +919,90 @@ describe('Facet v1 Test Suites', () => {
...
+      const index = [0, 1, 2];
+      function setupSelectedMulitpleFacets() {
+        setupBreadboxWithFacet();
+        index.forEach((i: number) => {
+          selectIdleCheckboxValueAt(FacetSelectors, i);
+          cy.wait(TestFixture.interceptAliases.Search);
+        });
+      }
+
+      describe('verify rendering', () => {
```
Comment (ThibodeauJF) [inline packages/atomic/cypress/integration/facets/facet/facet.cypress.ts:994]: One thing that is very important, especially for date & numeric facet, is to ensure that the format of the breadcrumb value is correct

**Review by ThibodeauJF — APPROVED** [review]

**Review by samisayegh — APPROVED** [review]

File: packages/atomic/cypress/integration/facets/facet/facet.cypress.ts (hunk 2 above)
Comment (lvu285) [reply packages/atomic/cypress/integration/facets/facet/facet.cypress.ts:994]: added!

**Pull request merged**